### PR TITLE
Add the first Cross-Chain Bridge between Move and EVM.

### PR DESCRIPTION
**Pull request by Foorgoo**
1, Add the first Cross-Chain Bridge between Move and EVM.
2, Add more tokens on Starcoin.
3, Add more wallets on Sui and Aptos.
4, Add Starcoin IDE maintained by the Starcoin team.
5, Add the tutorials of Move and SmartContract Development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,8 +77,11 @@ Code written in Move.
...
 - [GAS](https://github.com/OLSF/libra/blob/main/language/diem-framework/modules/0L/GAS.move) - A token that instantiates the Diem standard above. Deployed on 0L.
 - [STC](https://github.com/starcoinorg/starcoin-framework/blob/main/sources/STC.move) - A token that instantiates the Starcoin standard above. Deployed on Starcoin.
+- [STAR](https://github.com/Elements-Studio/starswap-core/blob/master/sources/gov/STAR.move) - A governance token of Starswap dApp that powers the AMM+DEX ecosystem. Deployed on Starcoin.
+- [XUSDT](https://github.com/Elements-Studio/poly-stc-contracts/blob/master/src/modules/asset/erc20/XUSDT.move) - A mapped assets of USDT on Starcoin.
+- [XETH](https://github.com/Elements-Studio/poly-stc-contracts/blob/master/src/modules/asset/erc20/XETH.move) - A mapped assets of ETH on Starcoin.
 - [WEN stablecoin](https://github.com/wenwenprotocol/wen-protocol) - Deployed on Starcoin.
-- [FAI stablecoin](https://github.com/BFlyFinance/FAI) - Deployed on Starcoin.
+- [FAI stablecoin](https://github.com/BFlyFinance/FAI) - An algorithm based stable coin deployed on Starcoin.
```
Comment (jolestar) [inline README.md:84]: I think FAI is not `algorithm based`

Comment (Foorgoo) [reply README.md:84]: > I think FAI is not `algorithm based`

Not algorithm based? I took it from their github intro.Please correct it if it's not correct, thanks!

**Comment by sblackshear** [comment]
LGTM, just need some conflicts resolved!

File: README.md (hunk 1 above)
Comment (6543ngc) [reply README.md:84]: An over-collateralized stable coin is more accurate, we will correct it in our repo, thanks

Comment (Foorgoo) [reply README.md:84]: > An over-collateralized stable coin is more accurate, we will correct it in our repo, thanks

OK，I'll correct it.

**Comment by sblackshear** [comment]
Looks like there are a few broken links

**Comment by Foorgoo** [comment]
@6543ngc Has FLY's repository been deleted, causing the link to be wrong again.

**Comment by Foorgoo** [comment]
I checked again, it's not a broken link problem.please check, thanks. @sblackshear

**Comment by villesundell** [comment]
@Foorgoo Thanks for fixing the linting issue, the list now passes lint! :partying_face: However,there were two actual broken links (I left the correct addresses as review comments), once those are fixed, this will be merged.

**Pull request merged**